docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -3,912 +3,327 @@ This guide is maintained in the main Quarkus repository
-* Creating a reactive JAX-RS endpoint (asynchronous, streams...)
-* Using reactive database access
-* Interacting with other reactive APIs
+Quarkus is a _Reactive_ framework.
+Since the beginning, _Reactive_ has been an essential tenet of the Quarkus architecture.
+It offers many reactive features and offers a broad ecosystem.
+
+This guide is not an in-depth article about what _Reactive_ is and how Quarkus enables reactive architectures.
+If you want to read more about these topics, refer to xref:quarkus-reactive-architecture.adoc[], which provides an overview of the Quarkus reactive ecosystem.
+
+In this guide, we will get you started with some reactive features of Quarkus.
+We are going to implement a simple CRUD application.
+Yet, unlike in the xref:hibernate-orm-panache.adoc[Hibernate with Panache guide], it uses the reactive features of Quarkus.
+
+This guide will help you:
+
+* Bootstrapping a reactive CRUD application with Quarkus
+* Using Hibernate Reactive with Panache to interact with a database in a reactive fashion
+* Using RESTEasy Reactive to implement HTTP API while enforcing the reactive principle
+* Packaging and Running the application
 
 == Prerequisites
 
 To complete this guide, you need:
 
-* less than 15 minutes
-* an IDE
-* JDK 11+ installed with `JAVA_HOME` configured appropriately
+* around 15 minutes
+* your favorite IDE
+* Java 11+ installed with `JAVA_HOME` configured appropriately.
 * Apache Maven {maven-version}
 
-== Solutions
+NOTE: Verify that Maven is using the Java version you expect.
+If you have multiple JDKs installed, make sure Maven is using the expected one.
+You can verify which JDK Maven uses by running `mvn --version.`
 
-We recommend that you follow the instructions from <<bootstrapping-the-project,Bootstrapping project>> and onwards to create the application step by step.
+== Imperative vs. Reactive: a question of threads
 
-However, you can go right to the completed example.
+As mentioned above, in this guide, we are going to implement a reactive CRUD application.
+But you may wonder what the differences and benefits are in comparison to the traditional and imperative model.
 
-Download an {quickstarts-archive-url}[archive] or clone the git repository:
+To better understand the contrast, we need to explain the difference between the reactive and imperative execution models.
+It's essential to comprehend that _Reactive_ is not just a different execution model, but that distinction is necessary to understand this guide.
 
-[source,bash,subs=attributes+]
-----
-git clone {quickstarts-clone-url}
-----
+In the traditional and imperative approach, frameworks assign a thread to handle the request.
+So, the whole processing of the request runs on this worker thread.
+This model does not scale very well.
+Indeed, to handle multiple concurrent requests, you need multiple threads; and so your application concurrency is constrained by the number of threads.
```

Comment:
Perhaps it would help to add that Java Threads a mapped one to one with OS threads (at least until Loom arives) and the OS threads are what contrains the scalability of this model

Reply:
Fixed, added the reference to OS thread in the last sentence of this paragraph.